### PR TITLE
Update OpenAPI client again

### PR DESCRIPTION
**Pull request by julienduchesne**
Following https://github.com/grafana/grafana-openapi-client-go/pull/40, we don't need most of the "params" objects

### CODE REVIEW

**Comment by julienduchesne** [comment]
Cloud instance tests failing due to some SLO stuff. Presumably fixed by https://github.com/grafana/terraform-provider-grafana/pull/1186

**Comment by inkel** [comment]
Should we wait for #1186 to be merged before merging this one?

**Comment by julienduchesne** [comment]
> Should we wait for #1186 to be merged before merging this one?

No, as long as the other tests pass, I think it's fine. In this PR, I'm not modifying anything related to the cloud instance, so there's no reason why they should fail from this PR

**Comment by julienduchesne** [comment]
> Should we wait for #1186 to be merged before merging this one?

It's merged now

**Review by inkel — APPROVED** [review]

**Pull request merged**
